web/src/components/CreateStorageServiceDialog.tsx

```diff
@@ -170,6 +171,17 @@ const CreateStorageServiceDialog: React.FC<Props> = (props: Props) => {
           onChange={(e) => setPartialS3Config({ secretKey: e.target.value })}
           fullWidth
         />
+        <Typography className="!mb-1" level="body2">
+          Bucket
+          <span className="text-sm text-gray-400 ml-1">(Storage Path)</span>
+        </Typography>
+        <Input
+          className="mb-2"
+          placeholder="Path"
+          value={s3Config.path}
+          onChange={(e) => setPartialS3Config({ path: e.target.value })}
+          fullWidth
+        />
```

Comment:
I think the setting of the `path` should be after the `bucket name`,  WDYT?

Reply:
Make sense, let me change that.